src/main/java/org/team1540/robot2025/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,47 @@
+package org.team1540.robot2025.subsystems.arm;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class ArmConstants {
+    public static final double GEAR_RATIO = 28;
+    public static final double ARM_MOMENT_OF_INERTIA = 0; // TODO:
+    public static final double ARM_LENGTH_METERS = 0.411; // TODO: check please
+
+    // arm straight down is 0, min angle is 30
+    public static final Rotation2d MIN_ANGLE = Rotation2d.fromDegrees(30);
+    public static final Rotation2d MAX_ANGLE = Rotation2d.fromDegrees(330);
```

Comment:
For kG's gravity compensation to work, the arm should be parallel to the floor at 0.

Reply:
or just change it to a sine profile

Reply:
The internal TalonFX kG only uses a cosine profile, which means we would have to swap to using an external FF that runs at a lower frequency. It'd be simpler and faster to keep it all on the Talon.